internal/vm.h

```diff
@@ -83,6 +83,23 @@ void rb_check_stack_overflow(void);
 extern uint64_t rb_vm_insns_count;
 #endif
 
+extern bool rb_free_on_shutdown;
+
+void rb_free_default_rand_key(void);
```

Comment:
Can we move these function declarations to the header file for the respective C files rather than placing them all in `vm.h`?